cylc/flow/scripts/cylc.py

```diff
@@ -33,15 +34,30 @@
 )
 
 
+def get_version(long=False):
+    """Return the Cylc Flow version and top level installation path.
```

Comment:
Top level installation path only if long is True?

Reply:
Yep, I'll clarify that.